packages/applications/document-builder/src/candidature/attestation/attestation.saga.ts

```diff
@@ -116,6 +116,10 @@ export const register = () => {
           });
           return;
         }
+        if (!candidature.notification.attestation) {
+          logger.info(`La candidature n'a pas d'attestation existante.`, { identifiantProjet });
```

Comment:
Pour aligner avec au dessus : 

```suggestion
          logger.info(`L'attestation ne sera pas régénérée car la candidature n'a pas d'attestation existante.`, { identifiantProjet });
```